### PR TITLE
[Prices] Adding 50 tokens to price feed

### PR DESCRIPTION
**Pull request by bh2smith**
Based on [this query](https://dune.com/queries/1317238?d=1&category=abstraction&namespace=prices&table=usd_latest&blockchains=arbitrum+avalanche_c+bnb+ethereum+gnosis+optimism+solana) which shows the list of all tokens traded on cow_protocol that have missing prices. We wrote a script to join these with Coin Paprika's [coins-API](https://api.coinpaprika.com/v1/coins) and [ethereum contracts](https://api.coinpaprika.com/v1/contracts/eth-ethereum).

This found a total of 436 matches (see logs from script)

```sh
Getting Coin Paprika token list
Excluded address for 4779 entries out of 44793
Loaded 2564 coins from Coin Paprika
Fetched 905 traded tokens from Dune without prices
Found 436 matches
Results written to missing-prices.txt
```

Related PR used to generate results: https://github.com/cowprotocol/data-misc/pull/15

The complete file is here, but I am starting with the top 50 to see how it goes.

[missing-tokens.txt](https://github.com/duneanalytics/spellbook/files/9646108/missing-tokens.txt)

### CODE REVIEW

File: models/prices/prices_tokens.sql [hunk 1]
```diff
@@ -589,6 +589,57 @@ VALUES
...
+    ("spool-spool-dao-token", "ethereum", "SPOOL", "0x40803cea2b2a32bda1be61d3604af6a814e70976", 18),
+    ("upi-pawtocol", "ethereum", "UPI", "0x70d2b7c19352bb76e4409858ff5746e500f2b67c", 18),
+    ("media-media-network", "ethereum", "MEDIA", "0xdb726152680ece3c9291f1016f1d36f3995f6941", 6),
+    ("inuinu-inu-inu", "ethereum", "INUINU", "0xc6bdb96e29c38dc43f014eed44de4106a6a8eb5f", 18),
+    ("seth2-seth2", "ethereum", "SETH2", "0xfe2e637202056d30016725477c5da089ab0a043a", 18),
+    ("xi-xi-token", "ethereum", "XI", "0x295b42684f90c77da7ea46336001010f2791ec8c", 18),
+    ("bank-bankless-dao", "ethereum", "BANK", "0x2d94aa3e47d9d5024503ca8491fce9a2fb4da198", 18),
+    ("eth2x-fli-eth-2x-flexible-leverage-index", "ethereum", "ETH2X-FLI", "0xaa6e8127831c9de45ae56bb1b0d4d4da6e5665bd", 18),
+    ("floor-floordao", "ethereum", "FLOOR", "0xf59257e961883636290411c11ec5ae622d19455e", 9),
+
```
Comment (bh2smith) [inline models/prices/prices_tokens.sql:642]: Also added this line separating blockchains back.

**Review by soispoke — APPROVED** [review]
lgtm

**Comment by bh2smith** [comment]
If this is alright and works on your end without too much hassle, I would proceed in going with 100 for the next round. I could also progressively add more here to this PR so not to pollute the open PRs with identical stuff.

**Comment by bh2smith** [comment]
So, should I wait a day before I add more tokens here?

We've got ~400 for the first (preliminary) pass though the token lists (i.e. those which were naively uniquely matchable). I have a second pass through the missing token list meant to capture those which require a bit more manual intervention.

Should we aim to put the first 436 directly in this PR?

**Comment by soispoke** [comment]
No let's wait and see how this goes for the 50 new tokens, and then merge the rest if everything looks good tmw

**Comment by bh2smith** [comment]
But wait... if this is merged already I can't keep adding to this PR. I suppose I'll make a new one tomorrow with the remaining tokens from the list.

**Comment by soispoke** [comment]
yeah you're right, that sounds good :)

**Comment by jeff-dude** [comment]
@bh2smith thank you for the detailed steps to build out the list for prices which match the API results. this has potential to be helpful for us in automation around prices PRs, with some added complexities once you consider more than ethereum blockchain, but in the right direction. are you good with us potentially looking closer at your code and reuse it?

**Comment by bh2smith** [comment]
> this has potential to be helpful for us in automation around prices PRs, with some added complexities once you consider 

Yes that sounds excellent. In that case I will actually clean up the code (to use objects/dataclasses instead of raw dictionaries). Would be happy if you wanted to use it in your backend. Let me know if/when you want to look into it again and I will ensure it is strictly typed.

**Pull request merged**